explainaboard/utils/span_utils.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+
+def get_spans_from_bio(seq):
```

Comment:
I actually haven't figured out the content of `seq`  especially after reading the docstring, should it be list[str], such as ["O", "B-PER", "I-PER"]?

Reply:
Thanks! This doc was legacy doc from before, but I made it clearer (and also added a test checking the desired behavior)

Reply:
nice! Feel free to merge this PR!